### PR TITLE
add funcionality to read a sql statement from a file, if path provided

### PR DESCRIPTION
**Pull request by talevy**

### CODE REVIEW

File: lib/logstash/inputs/jdbc.rb [hunk 1]
```diff
@@ -81,6 +83,9 @@ def register
     elsif File.exists?(@last_run_metadata_path)
       @sql_last_start = YAML.load(File.read(@last_run_metadata_path))
     end
+
+    # load statement from file, if filepath passed.
+    @statement = File.read(@statement) if File.exists?(@statement)
```
Comment (suyograo) [inline lib/logstash/inputs/jdbc.rb]: can we add an explicit `statement_file` config to take the configuration from a SQL file? Much cleaner IMO

**Comment by talevy** [comment]
@suyograo updated with `statement_filepath`

File: lib/logstash/inputs/jdbc.rb [hunk 2]
```diff
@@ -38,6 +39,14 @@
 #     schedule => "* * * * *"
 #   }
 # }
+#
+# #### statement vs statement_filepath
```
Comment (suyograo) [inline lib/logstash/inputs/jdbc.rb]: `====` instead of `####` for asciidoc :)

**Comment by suyograo** [comment]
LGTM

**Pull request merged**